tests/tape/test_qscript.py

```diff
@@ -192,7 +192,7 @@ def test_update_par_info_update_trainable_params(self):
         assert p_i[4] == {"op": ops[2], "op_idx": 2, "p_idx": 0}
         assert p_i[5] == {"op": ops[3], "op_idx": 3, "p_idx": 0}
         assert p_i[6] == {"op": ops[3], "op_idx": 3, "p_idx": 1}
-        assert p_i[7] == {"op": m[0].obs, "op_idx": 0, "p_idx": 0}
+        assert p_i[7] == {"op": m[0].obs, "op_idx": 4, "p_idx": 0}
```

Comment:
Just leaving a note here in case this change is reverted.

Reply:
Reverted.